extensions/wikia/AdEngine/js/slot/floatingMedrec.js

```diff
@@ -57,21 +57,21 @@ define('ext.wikia.adEngine.slot.floatingMedrec', [
 				});
 			}
 
-			if ($win.scrollTop() > startPosition && $win.scrollTop() < stopPosition) {
-				$adSlot.css({
-					position: 'fixed',
-					top: globalNavigationHeight + margin + 'px',
-					visibility: 'visible'
-				});
-			}
-
-			if ($win.scrollTop() >= stopPosition) {
-				$adSlot.css({
-					position: 'absolute',
-					top: stopPosition - startPosition + 'px',
-					visibility: 'visible'
-				});
-			}
+			// if ($win.scrollTop() > startPosition && $win.scrollTop() < stopPosition) {
```

Comment:
Instead of commenting out let's not load the file for the experiment and if it's not possible wrap it with AB test conditions.